crypto/x509/x509_vfy.c

```diff
@@ -3440,6 +3444,29 @@ static int check_key_level(X509_STORE_CTX *ctx, X509 *cert)
     return EVP_PKEY_security_bits(pkey) >= minbits_table[level - 1];
 }
 
+/*
+ * Check whether the public key of ``cert`` does not use explicit params
+ * for an elliptic curve.
+ *
+ * Returns 1 on success, 0 otherwise.
```

Comment:
Maybe tri-state return?

Reply:
Done

Reply:
Nit: comment here needs updating for tri state

Reply:
ok